image/histogram/driver.py

```diff
@@ -0,0 +1,2 @@
+import main1.image_histogram
```

Comment:
The driver should do the input and visualisation.

Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.